1.7/administration/upgrading.md

```diff
@@ -51,7 +51,13 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 ### DC/OS Masters
 
-Identify your Mesos leader node. This node should be the last master node that you upgrade. Proceed with upgrading every master node using the following procedure. When you complete each upgrade, monitor the logs to ensure the unit has re-joined the cluster and completed reconciliation.
+Identify the ZooKeeper leader among the masters. This node should be the last master node that you upgrade. You can determine whether a master node is a ZooKeeper leader by sending the `stat` command to the ZooKeeper client port.
+
```

Comment:
This assumes that the OS provides `echo` and `grep`, which are common utilities, and it doesn't use them with any platform-specific flags. This example should be portable among most distros, certainly all that we support. /cc @cmaloney